tests/models/dsheetpiling/test_dsheetpiling_acceptance.py

```diff
@@ -92,12 +89,14 @@
     Soil,
     SoilType,
 )
+from pydantic.color import Color
+from teamcity import is_running_under_teamcity
```

Comment:
If you removed all uses of this package then delete it

Reply:
I will check it again 👍

Reply:
I removed all unusued references.